src/third_party/dartino/gc_metadata.cc

```diff
@@ -63,6 +63,11 @@ void GcMetadata::set_up_singleton() {
   // page boundaries.
   metadata_ = reinterpret_cast<uint8*>(OS::grab_virtual_memory(null, metadata_size_));
 
+  if (metadata_ == null) {
+    printf("[Toit] ERROR Failed to allocate GC metadata\n");
```

Comment:
I don't think this is the conventional format for our messages.

```suggestion
    printf("[toit] ERROR: failed to allocate GC metadata\n");
```